app/routes/static.rb

```diff
@@ -51,7 +51,7 @@ class Static < Core
       get '/contribute' do
         erb :"site/contribute"
       end
-
+      
```

Comment:
Would you make sure to keep the trailing whitespace off of the ends of lines? I hate being nitpicky about it, but it ends up adding noise to the diffs, so I try to keep the commits free of it.